src/contractWrappers/KlerosWrapper.js

```diff
@@ -381,7 +381,8 @@ class KlerosWrapper extends ContractWrapper {
         rulingChoices: dispute[3].toNumber(),
         initialNumberJurors: dispute[4].toNumber(),
         arbitrationFeePerJuror: dispute[5].toNumber(),
-        state: dispute[6].toNumber()
+        state: dispute[6].toNumber(),
+        status: (await contractInstance.disputeStatus(disputeId)).toNumber()
```

Comment:
Wow i didn't even realize this existed. Do you think it is useful? They seem easily inferable to me but if you have a use for them go ahead and leave it

Reply:
DisputeState isn't really useful for the front end. You would need to make a call to getArbitratorData to be able to differentiate between open and appealable based on the period. Returning status makes things easier.

Reply:
👍